test/tools/runner/config.js

```diff
@@ -224,10 +230,10 @@ class NativeConfiguration {
 
   writeConcernMax() {
     if (this.topologyType !== TopologyType.Single) {
-      return { w: 'majority', wtimeout: 30000 };
+      return { writeConcern: { w: 'majority', wtimeout: 30000 } };
     }
 
-    return { w: 1 };
+    return { writeConcern: { w: 1 } };
   }
 
```

Comment:
Per @nbbeeken 's suggestion. The benefit of making this change should mean about half as many warnings during testing.